README.md

```diff
@@ -0,0 +1,9 @@
+# Jellyfin.media
+
+The official Jellyfin website.
+
+Uses the theme ["Evie" theme](https://github.com/anges244/evie), copyright (c) 2018 [Aggelos Gesoulis](https://twitter.com/anges244) - [Katerina Limpitsouni](https://twitter.com/ninalimpi), available under the MIT license.
+
+The header photo is by [freestocks.org](https://unsplash.com/photos/11SgH7U6TmI?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Unsplash](https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText), with minor edits.
```

Comment:
shouldn't we clean these `utm_source=` things?

Reply:
Hrm. Yeah, they’re link source tracking. I just took the attribution link as they gave it to me. Will do.